HubSpot.NET/Core/HubSpotBaseClient.cs

```diff
@@ -123,6 +126,41 @@ public T ExecuteMultipart<T>(string absoluteUriPath, byte[] data, string filenam
             return responseData;
         }
 
+        public async Task<T> ExecuteFileUpload<T>(string absoluteUriPath, byte[] data, string filename, Dictionary<string,string> parameters)  where T : new()
+        {
+            string path = $"{BaseUrl}{absoluteUriPath}";
+            
+            var httpClient = new HttpClient();
```

Comment:
might be a good idea to put this httpclient in a using block so it gets disposed accordingly